vocabularies/sampling-method.ttl

```diff
@@ -0,0 +1,890 @@
+@prefix smpmh: <http://linked.data.gov.au/def/sampling-method/> .
+@prefix gasmp: <http://pid.geoscience.gov.au/def/voc/ga/samplingmethod/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/sampling-method> a owl:Ontology ,
+        skos:ConceptScheme ;
+    dct:created "2019-06-03"^^xsd:date ;
+    dct:creator <http://linked.data.gov.au/org/gsq> ;
+    dct:modified "2020-06-03"^^xsd:date ;
+    dct:publisher <http://linked.data.gov.au/org/gsq> ;
+    skos:definition "The method of sample collection."@en ;
+    skos:hasTopConcept gasmp:airfall_dust_collector , 
+        gasmp:borehole_fluid_sampling , 
+        gasmp:borehole_logging_tool , 
+        gasmp:bottle_incubation , 
+        gasmp:camera , 
+        gasmp:chamber_incubation , 
+        gasmp:core_incubation_dark , 
+        gasmp:core_incubation_light , 
+        gasmp:current_meter , 
+        gasmp:drilling , 
+        gasmp:field_mapping_observation , 
+        gasmp:filtered_dust_collector , 
+        gasmp:filtration , 
+        gasmp:grainsize_separation , 
+        gasmp:marine_survey_observation , 
+        gasmp:marine_survey_sampling , 
+        gasmp:sediment_trap , 
+        gasmp:sonde , 
+        gasmp:still_photography , 
+        gasmp:surface_material_sampling , 
+        gasmp:undergound_mine_sampling , 
+        gasmp:underwater_camera , 
+        gasmp:unknown , 
+        gasmp:video_photography , 
+        gasmp:water_sampling ;
+    skos:prefLabel "Sampling Method"@en .
+
+<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
+    sdo:name "Geological Survey of Queensland" ;
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
+
+smpmh:unconsolidated_sediment_sampling a skos:Concept ;
+	dcterms:created "2020-06-03"^^xsd:date ;

```

Comment:
Turtle validator giving me **Undefined prefix "dcterms:" on line 48.**

Reply:
Good pick-up. Changed all dcterms to dct.